Engine/plugin/agsplugin.h

```diff
@@ -154,6 +154,8 @@ class IAGSEditor {
 #define PSND_OGGSTATIC  5
 #define PSND_MIDI       6
 #define PSND_MOD        7
+#define PSND_FLACSTREAM 8
+#define PSND_FLACSTATIC 9
```

Comment:
There's no need to make two separate constants here. The Static/Stream separation is remnant of old internal implementation, and should not have been a "type" ever in plugin api (possibly it should have been a separate parameter, but it's too late now).

Overall this type should not be even used in command, as engine should be trying to decode the sound using the extension as a hint first, and if that does not work, then trying all the available decoders.

Reply:
Good thing this is AGS4. It's not too late to do anything.
I'll add just one type then and forcepush again.